designs/virtualization/4.5/Scheduling/Scheduling-section/Scheduling-section.md

```diff
@@ -0,0 +1,18 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Scheduling Section
+
+Creating a new section in the VM details page, Overview tab
+('Scheduling': If, where and how this workload will be placed within the cluster's nodes)
+
+![Operator details page](img/vm-page-0-0.jpg)
```

Comment:
```suggestion
![Operator details page](img/VM-page-0-0.jpg)
```

Reply:
oh wow! this is why I couldn't see the image in Github! (I saw it only in my VSCode) Thanks!